woodwork/type_sys/inference_functions.py

```diff
@@ -115,17 +115,10 @@ def boolean_nullable_func(series):
         series_no_null = series.dropna()
         try:
             series_no_null_unq = set(series_no_null)
-            if series_no_null_unq in [
-                {False, True},
-                {True},
-                {False},
-            ]:
+            if series_no_null_unq.issubset({False, True}):
                 return True
-            series_lower = set(str(s).lower() for s in set(series_no_null))
-            if series_lower in [
-                set(boolean_list)
-                for boolean_list in config.get_option("boolean_inference_strings")
-            ]:
+            series_lower = set(str(s).lower() for s in series_no_null_unq)
```

Comment:
Just wondering, if the length of `series_no_null_unq` is greater than two, is there a scenario where we still need to do this computation?

Reply:
I can't comment on GitHub on this particular line in the MR, but I was wondering the following about null handling: 

On the parent function `boolean_func`, we call `boolean_nullable_func` and then check if there are any nulls -- `not series.isnull().any()`. I think we can leverage short circuiting and flip these two for potential speedup in boolean inference.